docs/collector.mssql.md

```diff
@@ -76,27 +76,27 @@ Name | Description | Type | Labels
 `windows_mssql_availreplica_resent_messages` | Number of Always On messages resent in the last second | counter | `mssql_instance`, `replica`
 `windows_mssql_availreplica_sends_to_replica` | Number of Always On messages sent to this availability replica per second | counter | `mssql_instance`, `replica`
 `windows_mssql_availreplica_sends_to_transport` | Actual number of Always On messages sent per second over the network to the remote availability replica | counter | `mssql_instance`, `replica`
-`windows_mssql_bufman_background_writer_pages` | Number of pages flushed to enforce the recovery interval settings | counter | `mssql_instance`
+`windows_mssql_bufman_background_writer_pages_total` | Number of pages flushed to enforce the recovery interval settings | counter | `mssql_instance`
 `windows_mssql_bufman_buffer_cache_hit_ratio` | Indicates the percentage of pages found in the buffer cache without having to read from disk. The ratio is the total number of cache hits divided by the total number of cache lookups over the last few thousand page accesses | counter | `mssql_instance`
-`windows_mssql_bufman_checkpoint_pages` | Indicates the number of pages flushed to disk per second by a checkpoint or other operation that require all dirty pages to be flushed | counter | `mssql_instance`
+`windows_mssql_bufman_checkpoint_pages_total` | Indicates the number of pages flushed to disk per second by a checkpoint or other operation that require all dirty pages to be flushed | counter | `mssql_instance`
 `windows_mssql_bufman_database_pages` | Indicates the number of pages in the buffer pool with database content | counter | `mssql_instance`
 `windows_mssql_bufman_extension_allocated_pages` | Total number of non-free cache pages in the buffer pool extension file | counter | `mssql_instance`
 `windows_mssql_bufman_extension_free_pages` | Total number of free cache pages in the buffer pool extension file | counter | `mssql_instance`
-`windows_mssql_bufman_extension_in_use_as_percentage` | _Not yet documented_ | counter | `mssql_instance`
+`windows_mssql_bufman_extension_in_use_as_percentage` | Percentage of the buffer pool extension paging file occupied by buffer manager pages | counter | `mssql_instance`
```

Comment:
As this is a percentage, this should be set to a Gauge in both the docs and collector code. That's not an issue you've introduced, but as the docs now match the metric name it would be good to correct the metric type :smiley: